manifests/charts/ztunnel/templates/configmap.yaml

```diff
@@ -0,0 +1,24 @@
+{{- define "mesh" }}
+  defaultConfig:
+    proxyMetadata: {}
+{{- end }}
+
+{{- $originalMesh := include "mesh" . | fromYaml }}
+{{- $mesh := mergeOverwrite $originalMesh .Values.meshConfig }}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: istio-ztunnel-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- .Values.labels | toYaml | nindent 4}}
+  annotations:
+    {{- .Values.annotations | toYaml | nindent 4}}
+data:
+  mesh: |-
+{{- if .Values.meshConfig }}
```

Comment:
ztunnel doesn't actually read meshConfig, it reads proxyConfig. Maybe we should have users pass `.Values.proxyConfig` directly to avoid confusion

Reply:
I agree. Revised to `.Values.proxyConfig`